scripts/manual_privacy_preference_v2_data_migration.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+import time
+
+from loguru import logger
+
+from fides.api.db.session import get_db_session
+from fides.config import CONFIG
+
+"""
+Run in shell with:
+python3 scripts/manual_privacy_preference_v2_data_migration.py -table privacypreferencehistory
```

Comment:
nit: not sure exactly how this will be executed to go against the customer DBs, but it may be nice to allow config options to be passed in via CLI arg? or at least without requiring `fides` to be installed in the python env on which they're executed? 

but may not be a big deal if we know we've got a shell that can run this with the required setup.

Reply:
Can you give me an example of what you mean here? What would that look like.

Reply:
oh just thinking something like 

```
python3 scripts/manual_privacy_preference_v2_data_migration.py -table privacypreferencehistory -database_uri [DATABSE_URI_HERE]
```

to make this script a bit more "standalone" (i.e. fewer dependencies), but i suppose we'd still probably want to be able to use the `get_db_session` helper from `fides` anyway, so my suggestion may not be a good one, i.e. may just be better to ensure we can execute the script from a python env with `fides` installed and with the right config env vars set, that shouldn't be too hard :)

Reply:
thank you for elaborating!